core/variant_call.cpp

```diff
@@ -2182,6 +2216,8 @@ void register_variant_methods() {
 	ADDFUNC1(PACKED_COLOR_ARRAY, NIL, PackedColorArray, remove, INT, "idx", varray());
 	ADDFUNC2R(PACKED_COLOR_ARRAY, INT, PackedColorArray, insert, INT, "idx", COLOR, "color", varray());
 	ADDFUNC1(PACKED_COLOR_ARRAY, NIL, PackedColorArray, resize, INT, "idx", varray());
+	ADDFUNC1R(PACKED_COLOR_ARRAY, BOOL, PackedColorArray, has, COLOR, "value", varray());
+	ADDFUNC0(PACKED_COLOR_ARRAY, NIL, PackedColorArray, sort, varray());
```

Comment:
Since PackedVectors are code types new methods should be added to the GDNative API as well: [gdnative_api.json](https://github.com/godotengine/godot/blob/master/modules/gdnative/gdnative_api.json), [packed_arrays.cpp](https://github.com/godotengine/godot/blob/master/modules/gdnative/gdnative/packed_arrays.cpp), [packed_arrays.h](https://github.com/godotengine/godot/blob/master/modules/gdnative/include/gdnative/packed_arrays.h)

Reply:
Thanks for the info, done.